src/mon/OSDMonitor.cc

```diff
@@ -11796,6 +11796,39 @@ bool OSDMonitor::prepare_command_impl(MonOpRequestRef op,
     wait_for_finished_proposal(op, new Monitor::C_Command(mon, op, 0, rs,
 					      get_last_committed() + 1));
     return true;
+  } else if (prefix == "osd pool fingerprint") {
```

Comment:
I think this should be a pool property set via 'osd pool set <poolname> <property> <value>'.  Look at something like 'compression_mode' as a model

Reply:
ok